package.json

```diff
@@ -3,10 +3,11 @@
     "version": "1.0.0",
     "description": "antalmanac.com",
     "scripts": {
-        "client-install": "cd ./client && npm install",
-        "server-install": "cd ./antalmanac-backend && if test -f package.json; then npm install; fi",
+        "client-install": "cd client && npm install ./",
+        "server-install": "cd antalmanac-backend && npm install ./",
         "dependency-install": "concurrently -n \"BACKEND,FRONTEND\" \"npm run server-install\" \"npm run client-install\"",
         "postinstall": "npm run dependency-install",
+        "preinstall": "git clone https://github.com/icssc-projects/antalmanac-backend.git 2>NUL &",
```

Comment:
Wait actually I don't think we can do a clone here since it should be added as a submodule.

I'm just going to write a small node script to handle this. Should be simpler than messing around with different bash commands.